Solutions/Azure Firewall/Analytic Rules/Azure Firewall - Port Sweep.yaml

```diff
@@ -25,26 +27,39 @@ query: |
   let RunTime = 1h;
   let StartRunTime = 1d;
   let EndRunTime = StartRunTime - RunTime;
-  let MinimumDifferentHostsThreashold = 200;
-  let ExcludedPorts = dynamic([80, 443]);
+  let MinimumDifferentHostsThreshold = 200;
+  let ExcludedPorts = dynamic([80 , 443]);
   let BinTime = 30s;
-  AzureDiagnostics
+  union isfuzzy=true(
+  AZFWApplicationRule
+  | where TimeGenerated  between (ago(StartRunTime) .. ago(EndRunTime))
```

Comment:
Again, the time clause is not needed here because historical context is not getting used or required.

Reply:
It has been updated.